src/type/type.js

```diff
@@ -193,7 +204,8 @@ export default function getType(graffitiModels, {name, description, fields}, pat
     if (type === 'Array') {
       if (subtype === 'Object') {
         const fields = subfields;
-        graphQLField.type = new GraphQLList(getType(graffitiModels, {name, description, fields}, newPath, rootType));
+        const connectionName = getConnectionName(graphQLType.name, name);
```

Comment:
These are not connections, but nested objects. Connections are based on references.